t/api-bucket-objects-list.t

```diff
@@ -153,3 +158,34 @@ expect_api_bucket_objects_list 'error no such bucket' => (
     expect_s3_errstr        => 'No such bucket error message',
 );
 
+expect_api_bucket_objects_list 'list objects on Google Cloud Storage (version 1)' => (
```

Comment:
Please small change in test case description. version 1 relates to S3 API call not to service

```suggestion
expect_api_bucket_objects_list 'list objects (version 1) on Google Cloud Storage' => (
```

Reply:
done